src/main/java/seedu/address/model/client/ClientCountryMatchesInputCountryPredicate.java

```diff
@@ -25,6 +25,15 @@ public boolean test(Client client) {
         return client.getCountry().equals(country);
     }
 
+    /**
+     * Returns string representation of user-input country.
+     *
+     * @return The string representation of the user-input country.
+     */
+    public String getInputCountryRepr() {
```

Comment:
```suggestion
    public String getInputCountryStringRepresentation() {
```
Better to have descriptive function names without abbreviations imo

Reply:
Ok sure